.github/workflows/assign_reviewer.yml

```diff
@@ -2,15 +2,20 @@ name: Assign Reviewer
 
 on:
   pull_request:
-    types: ["labeled"]
+    types:
+      - opened
+      - reopened
+      - edited
+      - review_request_removed
+      - labeled
 
 permissions:
   pull-requests: write
 
 jobs:
   assign_reviewer:
     runs-on: ubuntu-latest
-    if: github.event.label.name == 'dependencies' && github.event.pull_request.user.login == 'app/renovate'
+    if: contains(github.event.pull_request.labels.*.name, 'dependencies') && github.event.pull_request.requested_reviewers == null && (github.event.pull_request.user.login == 'renovate[bot]')
```

Comment:
I believe `app/renovate` is the right login name - note that the linked discussion compares `github.actor` and see the output of 

```sh
gh pr list --repo edgelesssys/constellation --json author | jq | grep renovate
```

Reply:
`github.actor` and `github.event.pull_request.user.login` both refer to the username of an entity.
Since all workflows triggered after https://github.com/edgelesssys/constellation/pull/3091 was merged were skipped, I would assume that `app/renovate` is not the correct value here.

Reply:
Actually, let me test if I can trigger these workflows on existing renovate PRs and see what the needed value is

Reply:
https://github.com/edgelesssys/constellation/actions/runs/9368485665/job/25790643346?pr=3138#step:2:8

Reply:
Thanks for checking, I'm surprised but convinced.

Reply:
We could also try matching on `user.type == Bot` (https://docs.github.com/en/webhooks/webhook-events-and-payloads#pull_request).